<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FirstWidget</class>
 <widget class="QWidget" name="FirstWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item alignment="Qt::AlignHCenter">
        <widget class="QPushButton" name="pb_Choose">
         <property name="text">
          <string>Choisir un scénario</string>
         </property>
         <property name="icon">
          <iconset resource="../ressources.qrc">
           <normaloff>:/icones/file</normaloff>:/icones/file</iconset>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QPushButton" name="pb_Begin">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Commencer la partie</string>
         </property>
         <property name="icon">
          <iconset resource="../ressources.qrc">
           <normaloff>:/icones/go</normaloff>:/icones/go</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item alignment="Qt::AlignHCenter">
        <widget class="QPushButton" name="pb_Load">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Charger une partie</string>
         </property>
         <property name="icon">
          <iconset resource="../ressources.qrc">
           <normaloff>:/icones/load</normaloff>:/icones/load</iconset>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QPushButton" name="pb_Save">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Sauvegarder une partie</string>
         </property>
         <property name="icon">
          <iconset resource="../ressources.qrc">
           <normaloff>:/icones/save</normaloff>:/icones/save</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Résumé:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="pte_summary">
     <property name="undoRedoEnabled">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../ressources.qrc"/>
 </resources>
 <connections/>
</ui>
